File: week_2/10_rheostat_control.py
from Arduino import Arduino
import time

potPin = 0
ledPin = 11

board = Arduino("9600", port = "COM3")
board.pinMode(ledPin, "OUTPUT")

while True:
    potValue = board.analogRead(potPin)
    brightness = potValue / 4
    board.analogWrite(ledPin, brightness)
    time.sleep(0.1)

File: week_2/10_rheostat_control.cpp
int potPin = 0;
int ledPin = 11;
int potValue = 0;
int brightness = 0;

void setup() {
    pinMode(ledPin, OUTPUT);
}

void loop() {
    potValue = analogRead(potPin);
    brightness = map(potValue, 0, 1023, 0, 255);

    digitalWrite(ledPin, brightness);
    delay(100);
}
